pkg/cloud/services/ec2/bastion.go

```diff
@@ -35,7 +35,8 @@ import (
 )
 
 const (
-	defaultSSHKeyName = "default"
+	defaultSSHKeyName         = "default"
+	defaultUsGovPartitionName = "us-gov"
```

Comment:
You already have this declared in `cmd/clusterawsadm/api/bootstrap/v1beta1/defaults.go`

Reply:
Thanks for pointing it out, removed duplicate.